src/views/CourtOrder.vue

```diff
@@ -318,6 +318,10 @@ export default class CourtOrderView extends Mixins(DateMixin, FilingMixin, Commo
 
   paymentErrorDialog = false
 
+  // Template Enums
+  readonly AuthorizedActions = AuthorizedActions
+  readonly IsAuthorized = IsAuthorized
```

Comment:
Not needed if not used in template.

I'll stop making this comment. Please check remaining files for same thing.